packages/caliper-core/lib/manager/monitors/monitor-docker.js

```diff
@@ -359,8 +362,13 @@ class MonitorDocker extends MonitorInterface {
                     watchItemStat.set('Name', container.name);
                     watchItemStat.set('Memory(max)', mem_stat.max);
                     watchItemStat.set('Memory(avg)', mem_stat.avg);
-                    watchItemStat.set('CPU%(max)', cpu_stat.max.toFixed(2));
-                    watchItemStat.set('CPU%(avg)', cpu_stat.avg.toFixed(2));
+                    if(this.options.hasOwnProperty('cpuUsageNormalization')){
```

Comment:
looks like you didn't make the suggested change ?

Reply:
I forgot to add the changes to the main docker mon itor file and only pushed the one of the test file my bad